fl4health/server/base_server.py

```diff
@@ -41,6 +45,16 @@ def __init__(
                 server side checkpointing based on some criteria. If none, then no server-side checkpointing is
                 performed. Defaults to None.
         """
+        if seed is None:
```

Comment:
Some comment as in the client code, we can encapsulate this logic into a function and I would suggest when seed is none then we don't set anything?